scripts/config_trilinos.sh

```diff
@@ -127,7 +127,7 @@ cmake \
     -D Trilinos_ENABLE_Teko:BOOL=OFF \
     -D Teko_ENABLE_TESTS:BOOL=OFF \
     -D Trilinos_ENABLE_MueLu:BOOL=ON \
-    -D Tpetra_INST_INT_LONG_LONG:BOOL=OFF \
+    -D Tpetra_INST_INT_LONG_LONG:BOOL=ON \
     -D Tpetra_INST_INT_LONG:BOOL=ON \
```

Comment:
I think this should be off if the other option is turned on? This is at least what I did.

Reply:
It has been a while since I last tried, but if both `LONG` and `LONG_LONG` can be on at the same time (I believe on Linux 64 bit they map to the same type), we can go with this.

Reply:
I think the second line (`...INT_LONG:BOOL`) is a leftover, so I removed it. However, I do not remember exactly.